MoppApp/MoppApp/ContainerActions.swift

```diff
@@ -44,18 +44,28 @@ extension ContainerActions where Self: UIViewController {
         topSigningViewController.present(landingViewController.importProgressViewController, animated: false)
         
         MoppFileManager.shared.importFiles(with: urls) { [weak self] error, dataFilePaths in
-        
+
             if landingViewController.fileImportIntent == .addToContainer {
                 landingViewController.addDataFilesToContainer(dataFilePaths: dataFilePaths)
             }
             else if landingViewController.fileImportIntent == .openOrCreate {
                 navController.setViewControllers([navController.viewControllers.first!], animated: false)
              
                 let ext = urls.first!.pathExtension
-                if ((ext.isAsicContainerExtension || ext == ContainerFormatPDF) &&
-                    landingViewController.containerType == .asic) &&
+                if landingViewController.containerType == nil {
+                    if ext.isAsicContainerExtension {
+                        landingViewController.containerType = .asic
+                    } else {
+                        landingViewController.containerType = .cdoc
+                    }
+                }
+                
+                if  (((ext.isAsicContainerExtension ||
```

Comment:
Badly readable code. Lines starting from different indentation and not clear where parentheses start and end.